test/api/api_test.go

```diff
@@ -113,10 +108,17 @@ func CompileBinary(t *testing.T, projectDir string, tempDir string) string {
 		binaryPath += `.exe`
 	}
 
+	// Envs
+	envs, err := env.FromOs()
+	assert.NoError(t, err)
+	envs.Set("TEMPLATES_API_BUILD_TARGET_PATH", binaryPath)
+	envs.Set("SKIP_API_CODE_REGENERATION", "1")
```

Comment:
Pri kompilovani API binarky potrebujeme ENVs z OS (path, gopath, ...).
Iba som to upravil z `os.Environ` na nas `env` balicek.